wallet/src/de/schildbach/wallet/ui/WalletActivity.java

```diff
@@ -417,9 +417,31 @@ protected void error(final int messageResId, final Object... messageArgs) {
     public boolean onCreateOptionsMenu(final Menu menu) {
         getMenuInflater().inflate(R.menu.wallet_options, menu);
         super.onCreateOptionsMenu(menu);
+
+        getMenuInflater().inflate(R.menu.wallet_options, menu);
```

Comment:
This is a duplicate of what is two lines above.

Reply:
I fixed this issue and rebased onto master.